js/lib/components/feedback/modal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import Modal from "react-modal";
+import Highlighter from "web-highlighter";
+
+Modal.setAppElement("body");
```

Comment:
If this does any significant computation or DOM manipulation, it shouldn't be called at the top-level.

Reply:
According to [react-modal's docs](https://reactcommunity.org/react-modal/accessibility/), this just ensures the page content is set as `aria-hidden` when the modal is open for screenreaders - so it shouldn't be expensive AFAIK (they also call it at the top-level in the docs).